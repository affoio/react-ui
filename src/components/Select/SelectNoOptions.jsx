import styled from 'styled-components';
import colors from '../../colors';

const SelectNoOptions = styled.div`
  width: 100%;
  color: ${colors.black[400]};
  text-align: center;
`;

export default SelectNoOptions;
