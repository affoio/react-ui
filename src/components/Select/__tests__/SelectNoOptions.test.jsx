import React from 'react';
import renderer from 'react-test-renderer';
import SelectNoOptions from '../SelectNoOptions';

describe('<SelectNoOptions', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SelectNoOptions />);

    expect(tree).toMatchSnapshot();
  });
});
