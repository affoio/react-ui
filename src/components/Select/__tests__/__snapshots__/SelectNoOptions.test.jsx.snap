// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SelectNoOptions renders correctly 1`] = `
.c0 {
  width: 100%;
  color: #bfbfbf;
  text-align: center;
}

<div
  className="c0"
/>
`;
